### DISCUSSION TITLE
Katas' description overflowing the container

### DISCUSSION THREAD

**Discussion by KonradLinkowski** [codewars/codewars.com#2814] [General]
`word-break: break-word` should probably be added in some places, because the text can overflow the container
https://www.codewars.com/kata/5fee4559135609002c1a1841

![screenshot of the text overflowing](https://user-images.githubusercontent.com/26126510/190666670-d09f864c-b457-463e-8e3f-8f5c5d1fdfc1.png)

**Comment by z-b-b** [comment]
Yes, you are right. contact Codewars: info@codewars.com. report the problem to them.

**Reply by hobovsky** [reply]
There's no need to contact Codewars at `info@codewars.com`, having the issue reported here is enough.

Ideally it would be raised as an issue on issues board, but this discussion can be turned into one.

**Comment by kazk** [comment]
Fixed. Please use the issue tracker next time. https://github.com/codewars/codewars.com/issues

**Reply by KonradLinkowski** [reply]
https://github.com/codewars/codewars.com/issues/2825
